Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -24,6 +23,14 @@ open class BaseMapView: UIView, CameraViewDelegate {
     /// List of completion blocks that need to be completed by the displayLink
     internal var pendingAnimatorCompletionBlocks: [PendingAnimationCompletion] = []
 
+    /// Pointer HashTable for holding camera animators
+    internal var cameraAnimators = NSHashTable<CameraAnimator>.weakObjects()
```

Comment:
I think we have an internal public api note/ticket somewhere for this but

rename this to `cameraAnimatorsHashTable`